.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: python
 jobs:
   include:
+    - python: "3.9"
+      env: TOXENV=py39
     - python: "3.8"
-      env: TOXENV=black,flake8
+      env: TOXENV=black,flake8  # as soon as higher version than 20.8b1 is released, we can move it to py39
```

Comment:
Should we say 

`as soon as black > 20.8b1 is released and we start using it, we can move these tests to the python 3.9 job`